README.md

```diff
@@ -4,132 +4,72 @@
-optimisations. It is good practice for operators to adapt to their
-environment and accelerate workloads accordingly.
-
-## Pure Python operators
-
-The [Python Operator Framework](https://pythonoperatorframework.io/) makes
-it easy to write an operator. The framework handles all the details of
-communication between integrated operators, so you can focus on your own
-[application lifecycle management](https://juju.is/docs/olm).
-
-Code sharing between operator publishers is simplified making it much faster
-to collaborate on distributed systems involving components from many
-different publishers and upstreams. Your operator is a Python event handler.
-Lifecycle management, configuration and integration are all events delivered
-to your charm by the framework.
-
-## Architecture
-
-The Juju client, server and agent are all written in Golang. The standard Juju
-packaging includes an embedded database for centralised logging and persistence,
-but there is no need to manage that database separately.
-
-Operators can be written in any language but we do encourage new authors to
-use the Python Operator Framework for ease of contribution, support and
-community participation.
-
-## Production grade
-
-The Juju server has built-in support for [high
-availability](https://juju.is/docs/olm/high-availability) when scaled
-out to three instances. It can monitor itself and grow additional instances
-in the event of failure, within predetermined limits. Juju supports backup,
-restore, and rolling upgrade operations appropriate for large-scale
-centralised enterprise grade management and operations systems.
-
-## Get started
-
-Our community hangs out at the [Charmhub discourse](https://discourse.juju.is/)
-which serves as a combination mailing list and web forum. Keep up with the news
-and get a feel for operator engineering and usage there. Get  the Juju CLI on
-Windows, macOS or Linux with the
-[install instructions](https://juju.is/docs/installing) and
-[try the tutorials](https://juju.is/docs/olm/tutorials). All you need is a small
-K8s cluster, or an Ubuntu machine or VM to run MicroK8s.
-
-Read the [documentation](https://juju.is/docs) for a comprehensive reference
-of commands and usage.
-
-## Contributing
-
-Check the [developer instructions](https://juju.is/docs/dev/how-to) to get
-started and follow our [code and contribution guidelines](CONTRIBUTING.md) to
-learn how to make code changes. File bugs in
-[Launchpad](https://bugs.launchpad.net/juju/+filebug) or ask questions on
-our [Freenode IRC channel](https://webchat.freenode.net/#juju), and
-[Mattermost](https://chat.charmhub.io/).
+Juju is an open source application orchestration engine that enables any application operation (deployment, integration, lifecycle management) on any infrastructure (Kubernetes or machines) at any scale (development or production) in the same easy way (typically, one line of code), through special operators called ‘charms’.
```

Comment:
As discussed, I will change "Kubernetes or machines" to "Kubernetes or otherwise".